common/scala/src/main/scala/whisk/core/entity/Identity.scala

```diff
@@ -32,10 +32,11 @@ import whisk.core.entitlement.Privilege
 case class UserLimits(invocationsPerMinute: Option[Int] = None,
                       concurrentInvocations: Option[Int] = None,
                       firesPerMinute: Option[Int] = None,
+                      disableActivationStore: Option[Boolean] = None,
```

Comment:
I do not like this - it's a brute force hammer and dramatically changes the user's experience: no logs, no activations. In any case, this should be treated as a quota with a limit on storing activations and then disabling all activation records would be equivalent to setting the quota down to 0.

Reply:
That definetly makes sense to me.
What would your proposal of an implementation? Should we have a per-minute limit for these logs and let the controller check if the current activation will be written or not?